lib/services/common.dart

```diff
@@ -10,8 +10,7 @@ import 'dart:math';
 // spread its traffic. For simplicity's sake, at the current time DartPad picks
 // one at launch and sticks with it until the window is closed.
 final serverUrls = [
-  'https://dart-services.appspot.com/',
-  'https://dart-services-0.appspot.com/',
+  'https://v1.api.dartpad.dev/',
```

Comment:
Since we're unlikely to need to spread traffic randomly over multiple URLs now that Cloud Run is in place, you should be able to rip out all this array/random/etc kruft that I added a while back.

Reply:
Done =)